cmd/soroban-cli/src/log/event.rs

```diff
@@ -0,0 +1,50 @@
+use crate::xdr;
```

Comment:
💡 I suggest writing files like this top-to-bottom, as it is it's more difficult for a reader to jump around. I noticed we've used this micro function jump around strategy in some areas of the CLI and it's harder for a reader unfamiliar with the code to jump in. I think we can get the same benefit of the small functions having names by adding a couple comments.

For example, change as it is:
<img width="1043" alt="Screenshot 2024-07-26 at 9 25 54 AM" src="https://github.com/user-attachments/assets/a5f9cf26-1ced-488b-9ae3-667c7d0efd84">

Vs top-to-bottom with comments:
<img width="1043" alt="Screenshot 2024-07-26 at 9 45 36 AM" src="https://github.com/user-attachments/assets/ddf1db08-42e3-4080-b608-e2278f14049e">